.npmrc

```diff
@@ -0,0 +1,4 @@
+registry=https://registry.npmjs.org/
+//npm.pkg.github.com/:_authToken=${GITHUB_PACKAGE_TOKEN}
```

Comment:
Is this needed?  Its a public repository, so we shouldn't need to include a token to get the package.

Reply:
I think this one is, `npm i` doesn't work locally without, even for 'public' packages
https://github.community/t/download-from-github-package-registry-without-authentication/14407